cmd/get-kustomize-build-base/README.md

```diff
@@ -0,0 +1,24 @@
+# `get-kustomize-build-base`
```

Comment:
Correct me if I'm wrong, but this component doesn't serve any purpose right now, right?
Because it replaces just part of `kustomize-build-dirs` logic. So, I understand, that with this PR you are just preparing the repository for actual changes.

Reply:
If so, I think this should be included in the PR description. 
It would also be worth to include information about your planning next steps- how are you finally planning to use this piece of work?
Right now, I honestly don't know what are we supposed to review

Reply:
> Correct me if I'm wrong, but this component doesn't serve any purpose right now, right? Because it replaces just part of `kustomize-build-dirs` logic. So, I understand, that with this PR you are just preparing the repository for actual changes.

it can be slotted in just before the call to `kustomize-build-dirs`: instead of `git diff --name-only ... | xargs kustomize-build-dirs` we can `git diff --raw ... | get-kustomize-build-base | xargs kustomize-build-dirs`. `kustomize-build-dirs` still has logic to find a build base, but since the values we pass it should already be build bases it should just work™

Reply:
> it can be slotted in just before the call to kustomize-build-dirs: instead of git diff --name-only ... 

So, it is the way we want to use this component as is now? Or, are you planning more changes?

Reply:
> So, it is the way we want to use this component as is now? Or, are you planning more changes?

The plan is to use it as-is (see updated description)

>  It would also be worth to include information about your planning next steps- how are you finally planning to use this piece of work?

Go shout, updated description